data/iatemplates/mysql_restarted.yml

```diff
@@ -0,0 +1,23 @@
+---
+rules:
+  - name: mysql_restarted
+    version: 1
+    summary: MySQL Restarted
+    expr: |-
+      mysql_global_status_uptime
+      < [[ .threshold ]]
+    params:
+      - name: threshold
+        summary: Seconds from the last restart
+        unit: "sec"
+        type: int
```

Comment:
i guess this was missed in the review but we only support float params https://github.com/percona-platform/platform/blob/main/pkg/alert/type.go#L16

as a result I can't parse this file in managed cc @AlekSi

Reply:
for now I have changed it to float in my PR to make the tests pass https://github.com/percona/pmm-managed/pull/558/files#r535992934